src/prefs.c

```diff
@@ -1633,8 +1633,14 @@ static gboolean prefs_dialog_key_press_response_cb(GtkWidget *dialog, GdkEventKe
 												   gpointer data)
 {
 	GeanyKeyBinding *kb = keybindings_lookup_item(GEANY_KEY_GROUP_HELP, GEANY_KEYS_HELP_HELP);
+	guint state, keyval;
+
+	if (event->keyval == 0)
+		return FALSE;
+
+	keybindings_get_normalised_event(event, &state, &keyval);
```

Comment:
One should probably check for the "legacy" keybindings with extra modifiers here too but I'm not sure the code should be uglyfied because of that. It's just a single keybinding used in a single specific case so I think it's not worth it.